src/create.rs

```diff
@@ -267,6 +269,11 @@ impl<'a> Builder<'a> {
         self
     }
 
+    pub fn target_bin_dir(&mut self, p: Option<&'a str>) -> &mut Self {
```

Comment:
Could you please add a brief doc comment for this method?

Reply:
done